src/docs/Components/Form/Datepicker/code.mdx

```diff
@@ -45,14 +61,26 @@ In addition, the `input` element must also have the `mc-text-input` class.
 
 <Preview path="default" />
 
+### Variations
+
+You can use any of the existing text-input component variations.
```

Comment:
`You may use all existing text-input component variations`

Reply:
fixed